### PR TITLE
Sqlentities

### PR DESCRIPTION
**Pull request by serghalak**
merge entities and V1__ and V2 migration and new module sandbox-service

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,6 +25,11 @@
 	</properties>
 	<dependencies>
 
+<!--		<dependency>-->
```
Comment (AndriiHolovianko) [inline pom.xml:28]: Please remove commented code

File: sandbox-data/src/main/java/com/exadel/sandbox/model/location/Location.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    private String street;
+
+    @Column(name = "number")
+    private String number;
+
+    @ManyToOne
+    @JoinColumn(name = "city_id")
+    private City city;
+
+//    @ManyToMany(mappedBy = "locations")
```
Comment (AndriiHolovianko) [inline sandbox-data/src/main/java/com/exadel/sandbox/model/location/Location.java]: Please remove commented code

File: sandbox-data/src/main/java/com/exadel/sandbox/model/user/Role.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+@Entity
+@Table(name = "role")
+public class Role extends BaseEntity {
+
+    @Column(name = "name")
+    private String name;
+
+    @OneToMany(mappedBy = "role")
+    private Set<User> users=new HashSet<>();
```
Comment (AndriiHolovianko) [inline sandbox-data/src/main/java/com/exadel/sandbox/model/user/Role.java:18]: Please format code properly

File: sandbox-data/src/main/java/com/exadel/sandbox/model/location/Location.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
+package com.exadel.sandbox.model.location;
+
+import com.exadel.sandbox.model.BaseEntity;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "location")
```
Comment (AndriiHolovianko) [inline sandbox-data/src/main/java/com/exadel/sandbox/model/location/Location.java:9]: Please introduce constants, see expamle https://www.baeldung.com/jpa-entity-table-names

**Review by AndriiHolovianko — APPROVED** [review]

**Comment by RosKokos** [comment]
Approved

**Pull request merged**